src/components/registration-view/registration-view.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+
+export function RegistrationView(props) {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [email, setEmail] = useState('');
+  const [birthday, setBirthday] = useState('');
+
+  const handleRegister = (e) => {
+    e.preventDefault();
+    console.log('form submitted');
+    /* props.onRegister(false); */
+  };
+
+  return (
+    <form onSubmit={handleRegister}>
```

Comment:
You have a formSubmit function and an onclick handler on the submit button. Choose one and use that. The onSubmit sends the form data in the function payload though. And with onclick, you have to grab the data from component state.